### PR TITLE
oded/add-to-exercise-util

### PR DESCRIPTION
**Pull request by odeds**

### CODE REVIEW

File: src/components/znkExercise/utility/service/znkExerciseUtilitySrv.js [hunk 1]
```diff
@@ -37,6 +37,25 @@
 
                 return !!isCorrect;
             };
+
+            ZnkExerciseUtilitySrv.setQuestionsGroupData = function (questions, groupData, playedAudioArticles) {
+                var groupDataMap = {};
+                playedAudioArticles = playedAudioArticles || [];
```
Comment (igormas) [inline src/components/znkExercise/utility/service/znkExerciseUtilitySrv.js]: Please remove since it belong only to toefl.

Comment (odeds) [inline src/components/znkExercise/utility/service/znkExerciseUtilitySrv.js]: done

File: src/components/znkExercise/utility/service/znkExerciseUtilitySrv.js [hunk 2]
```diff
@@ -37,6 +37,23 @@
...
+            ZnkExerciseUtilitySrv.setQuestionsGroupData = function (questions, groupData) {
+                var groupDataMap = {};
+
+                angular.forEach(groupData, function (group) {
+                    groupDataMap[group.id] = group;
+                });
+
+                angular.forEach(questions, function (question) {
+                    if (!groupDataMap[question.groupDataId]) {
+                        $log.debug('Group data is missing for the following question id ' + question.id);
```
Comment (assafshp) [inline src/components/znkExercise/utility/service/znkExerciseUtilitySrv.js]: all questions should have group data?

Comment (odeds) [inline src/components/znkExercise/utility/service/znkExerciseUtilitySrv.js]: mmm, i think it's like in toefl when there's no groupData it's just log that group data is missing

**Pull request merged**